client/state/gravatar-status/selectors.js

```diff
@@ -1,11 +1,15 @@
+import { getCurrentUserId } from 'calypso/state/current-user/selectors';
+
+import 'calypso/state/gravatar-status/init';
+
 /**
  * Returns true if we're currently uploading a Gravatar
  *
  * @param {object} state - The state
  * @returns {boolean} - If uploading a Gravatar
  */
 export function isCurrentUserUploadingGravatar( state ) {
-	return state.currentUser.gravatarStatus.isUploading ?? false;
+	return state.gravatarStatus.isUploading ?? false;
```

Comment:
Didn't notice this one before: the state is always a valid boolean so the `?? false` bit can be removed.